libcontainer/rootfs_linux.go

```diff
@@ -98,6 +98,15 @@ func prepareRootfs(pipe io.ReadWriter, iConfig *initConfig) (err error) {
 		return newSystemErrorWithCausef(err, "changing dir to %q", config.Rootfs)
 	}
 
+	s := iConfig.SpecState
+	if s != nil {
+		s.Pid = unix.Getpid()
+		s.Status = "creating"
```

Comment:
Are these status strings really not defined as constants anywhere?

Reply:
Good catch! I'll just add a const struct somewhere!

Reply:
Resolved here: https://github.com/opencontainers/runc/pull/2229/files#diff-ec0220f977914208b5bf7c1ff362365fR248-R253

Reply:
Also added a PR to the runtime spec here: https://github.com/opencontainers/runtime-spec/pull/1046/files